CRM/Report/Form.php

```diff
@@ -4672,6 +4680,10 @@ public function getPermissionedFTQuery(&$query, $alias = NULL, $return = FALSE)
     }
     $financialTypes = NULL;
     CRM_Financial_BAO_FinancialType::getAvailableFinancialTypes($financialTypes);
+    $this->financialTypes = $financialTypes;
```

Comment:
Something like this is probably shared on multiple reports so the only shared parent would probably be CRM_Core_Form for all the reports